test/integration/utility.cc

```diff
@@ -19,7 +19,7 @@
 
 #include "test/common/upstream/utility.h"
 #include "test/mocks/stats/mocks.h"
-#include "test/mocks/upstream/mocks.h"
+#include "test/mocks/upstream/cluster_info.h"
```

Comment:
also don't see a change here in the build rule. is it already scoped to only include cluster_info_mocks?